modin/pandas/test/utils.py

```diff
@@ -203,6 +203,11 @@
 test_data_categorical_values = list(test_data_categorical.values())
 test_data_categorical_keys = list(test_data_categorical.keys())
 
+test_data_categorical_dataframe = {
+    "A": pandas.Categorical([1, np.NaN]),
+    "B": pandas.Categorical([np.NaN, 2]),
```

Comment:
the essential problem of #1628 was that categorical data couldn't be split into partitions properly, let's the amount of rows, so they won't fit into a single partition. Default rows size in tests is 256, with this size they fully fill all of the 4 partitions used in tests.

Reply:
Done.